backend/controllers/logController.js

```diff
@@ -7,14 +7,13 @@ const HEADERS = [
   'TYPE',
   'REASON',
   'OBJECT',
-  'MESSAGE'
+  'MESSAGE',
 ];
 
-const getHeadersIndices = array => (
-  HEADERS.map((header) => array.indexOf(header))
-);
+const getHeadersIndices = (array) =>
```

Comment:
My preference here would to be add in whatever comment ignores the linter for a line (I think `/* eslint-disable */`) so that this functions has opening and closing parens .